harness/tests/experiment/fixtures/pytorch_onevar_model.py

```diff
@@ -229,6 +229,25 @@ def build_validation_data_loader(self) -> torch.utils.data.DataLoader:
             raise ValueError(f"unknown dataloader_type: {self.hparams['dataloader_type']}")
 
 
+class OneVarApexTrial(OneVarTrial):
```

Comment:
OneVarApexTrial is no longer used, right?

Reply:
Ah, correct!